pkg/api/store/cluster/cluster_store.go

```diff
@@ -236,18 +236,25 @@ func loadDataFromTemplate(clusterTemplateRevision *v3.ClusterTemplateRevision, c
 	dataFromTemplate["description"] = convert.ToString(data[managementv3.ClusterSpecFieldDisplayName])
 	dataFromTemplate[managementv3.ClusterSpecFieldClusterTemplateID] = ref.Ref(clusterTemplate)
 	dataFromTemplate[managementv3.ClusterSpecFieldClusterTemplateRevisionID] = convert.ToString(data[managementv3.ClusterSpecFieldClusterTemplateRevisionID])
-	dataFromTemplate[managementv3.ClusterSpecFieldClusterTemplateAnswers] = data[managementv3.ClusterSpecFieldClusterTemplateAnswers]
 
 	dataFromTemplate = transposeNameFields(dataFromTemplate, clusterConfigSchema)
 
 	//Add in any answers to the clusterTemplateRevision's Questions[]
-	answers := convert.ToMapInterface(data[managementv3.ClusterSpecFieldClusterTemplateAnswers])
-	allAnswers := convert.ToMapInterface(answers["values"])
+	allAnswers := convert.ToMapInterface(convert.ToMapInterface(data[managementv3.ClusterSpecFieldClusterTemplateAnswers])["values"])
```

Comment:
q: why do we convert to map twice?

Reply:
Because  it is a nested map structure. Example:
```
"answers" = {
           "values": {
	            "dockerRootDir": "/var/lib/docker123",
	            "rancherKubernetesEngineConfig.ignoreDockerVersion":
	            "false"
            }
          }
```